repose-aggregator/tests/performance-tests/src/performanceTest/resources/roles/gatling/tasks/install_gatling.yml

```diff
@@ -55,10 +75,8 @@
     mode: 0644
     backup: yes
 
-- name: Copy the Repose library for Gatling
+- name: Copy the Repose libraries and dependencies
```

Comment:
@Mario-Lopez mentioned moving this task from the `install_gatling`task collection to the `setup_test` task collection, and that seems fair to me.

Reply:
👍